inc/cache_enabler.class.php

```diff
@@ -141,8 +141,7 @@ function( $post_id ) {
             array(
                 __CLASS__,
                 'on_upgrade_hook',
-            )
```

Comment:
Please make the priority and argument integers on new lines (copy the format of the other `add_action` functions):

```
add_action(
    'upgrader_process_complete',
    array(
        __CLASS__,
        'on_upgrade_hook',
    ),
    10,
    2
);
```